packages/web-components/src/stories/timeline.stories.mdx

```diff
@@ -0,0 +1,379 @@
+            playhead: '2021-02-01T04:00:00Z'
+        }}
+    >
+        {Default.bind()}
+    </Story>
+</Canvas>
+
+## API
+
+<ArgsTable of="rux-timeline" />
+
+## Timeline
+
+The Timeline component is responsible for
+
+-   creating and manipulating the grid used to display events.
+-   rendering the appropriate labels for **Ruler**
+-   controlling the position of the playhead
+
+It is composed of the following subcomponents:
+-  Track
+-  Time Region
+-  Ruler
+
+## Zoom
+
+The width between each time interval can be controlled by the `zoom` property. By default, it starts at 1 and can be incremented. 
+The column width is automatically calculated based off of the timeline's current interval (days/hours/etc).
+You should consider your timeline's range and set an appropriate default zoom level so that your time intervals can fill the entire viewport.
+
+If your timeline spans multiple days, you will also want to consider manually controlling the interval. For example, if your timeline is set to days,
+you will probably want your interval to change to hours as your zoom level increases.
+
+### Setting the Time Regions
+
+To add any number of Time Regions, you can simply add them as children to any Track element:
+
+```
+<rux-track>
+    <div slot="label">
+        Region1
+    </div>
+    <rux-time-region
+        start="2021-02-01T00:00:00Z"
+        end="2021-02-01T00:30:00Z"
+    >
+        Region1
+    </rux-time-region>
+    <rux-time-region
+        start="2021-02-01T03:00:00Z"
+        end="2021-02-01T05:30:00Z"
+    >
+        Region2
+    </rux-time-region>
+</rux-track>
+```
+
+### Enabling the Ruler
+
+To enable the bottom ruler, make use of the `ruler` slot by passing in a `rux-ruler` within a `rux-track`:
+
+```
+<rux-track slot="ruler">
+  <rux-ruler></rux-ruler>
+</rux-track>
+```
+
+
+### Enabling Vertical Scrolling
+
+If your Timeline contains too many rows to fit into your viewport, you can make use of the `time-region-container` CSS Shadow Part and set a height using CSS.
+This will enable the overflow: scroll.
+
+For example: 
+
+```
+rux-timeline::part(time-region-container) {
+    height: 400px;
+}
+```
+
+<Story name="Track">{''}</Story>
+
+## Track
+
+Tracks are responsible for displaying content in horizontal lanes across the Timeline.
+They can include **Time Regions** or the **Ruler** component. 
+They are composed of a label section and a time region section.
+
+<ArgsTable of="rux-track" />
+
+### Setting the Label
+
+To set the label of a Region, you can use the `label` slot:
+
+```
+<rux-track>
+    <div slot="label">
+        Region1
+    </div>
+</rux-track>
+```
+
+## Time Region
+
+Time Regions represent single events in time. A `start` time and an `end` time are required. If a `start` and `end` time aren't provided, or an invalid range is provided,
+the component will throw an error not render. A valid `start` and `end` time must fall within the date range specified on the root **Timeline** component
```

Comment:
I think there's a missing 'and' - `the component will throw an error not render`